src/components/DataTypes/enum.tsx

```diff
@@ -1,17 +1,15 @@
 import type { JSONSchema7 as JSONSchema } from 'json-schema';
+import { ResolvableReturnType } from '../../common-types';
 import { randomString } from '../../utils';
+import { Operator } from '../Filters/SchemaSieve';
 import { getJsonDescription } from './utils';
 
-export const operators = {
-	is: {
-		getLabel: (_s: JSONSchema) => 'is',
-	},
-	is_not: {
-		getLabel: (_s: JSONSchema) => 'is not',
-	},
-};
+export const operators = (_s: JSONSchema) => ({
+	is: 'is',
+	is_not: 'is not',
+});
```

Comment:
Should we make an `operators` enum in a common file and just import the parts of that we need for each `operators` we create per DataType instead of manually re-entering each operator?

i.e. something like this instead of what we have:
```
export const operators = (_s: JSONSchema) => ([OPERATORS['is'], OPERATORS['is_not']].toObject())
```

Forgive the pseudocode but hopefully the above gets the idea across ^

Reply:
This was my thought as well, but this is not double because of the Object type. We need all dataTypes to have the same exact method to get operators.  Until we have the object exception for tags, we will have to stick with this solution unfortunatelly